lib/plugins/create/create.js

```diff
@@ -150,7 +150,7 @@ class Create {
     }
 
     if (!notPlugin) {
-      return BbPromise.try(() => {
+      return BbPromise.try(async () => {
```

Comment:
I belive this little function is not `async` as it only calls `fse.copySync` which is sync or `handleServiceCreationError` which is also a sync function. Am I missing something here?

Reply:
@pgrzesik good point. I would also avoid marking as _async_ functions passed to `BbPromise.try`.

Idea is to mark as _async_ any exported async functions that are part of some API, any inline functions it's fine to keep as the are.

And we need to especially careful with inline functions (e.g. marking constructor resolvers for `new Promise` introduces a bug). So by rule I would reject any _async_ on inline functions

Reply:
Thanks for your Feedback, I will work on changes. @medikoo do you mean I should not only remove the async in line 153, but also the async in the constructor line 40?

Reply:
At line 40 it's fine, as it's exported function (not just internal for internal function use)